doc/mgr/dashboard.rst

```diff
@@ -978,7 +979,7 @@ commands to manage roles are the following:
 
   $ ceph dashboard ac-role-del-scope-perms <rolename> <scopename>
 
-To associate roles to users, the following CLI commands are available:
+To associate roles to users, the following commands are avilable:
```

Comment:
```suggestion
To associate roles to users, the following commands are available:

Reply:
My ignominy is without bound.  Rephrased.